style.css

```diff
@@ -429,3 +429,119 @@ p {
   width: 100%;
   margin: 10px 0;
 }
+
+.contact-form {
+  background-image: url(images/Contact\ me.jpg);
+  background-repeat: no-repeat;
+  width: 375px;
+  height: 779px;
+}
+
+.title-p {
+  background-image: url(images/Ellipse\ 3.png);
+  background-repeat: no-repeat;
+  background-position: 328px 5px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-me {
+  background-image: url(images/Ellipse.jpg);
+  background-repeat: no-repeat;
+  background-position: 83px 1px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 100px;
+  color: #fff;
+  margin: 25px;
```

Comment:
- [ ] [Optional] Instead of adding every shape in your background image in a separate `background-image` you can use one png or SVG file and try to position it right like the Figma design but take care of the path by adding`./` before it.

- [ ] [Optional] Please add the correct margins between input fields and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160884537-7335464c-481c-4148-885b-f0296645be4f.png)  | ![image](https://user-images.githubusercontent.com/68796232/160884633-e7e13f89-6aa1-427d-a137-1afd6b4b739c.png)  |